ci/kind/test-e2e-kind.sh

```diff
@@ -53,6 +54,7 @@ proxy=true
 endpointslice=false
 np=true
 coverage=false
+skiplist="none"
```

Comment:
Can it use empty string ""? otherwise it may be error prone in the future as you are using `strings.Contains` to search keyword.

Reply:
fixed